larpandora/LArPandoraInterface/LArPandoraInput.cxx

```diff
@@ -55,7 +56,7 @@ namespace lar_pandora {
 
     art::ServiceHandle<geo::Geometry const> theGeometry;
     auto const detProp = art::ServiceHandle<detinfo::DetectorPropertiesService const>()->DataFor(e);
-    const bool isDualPhase(theGeometry->MaxPlanes() == 2);
+    const bool isDualPhase(lar_pandora::LArPandoraHelper::IsDualPhase());
```

Comment:
I think the explicit lar_pandora namespace can be dropped here for consistency with the equivalent calls later in the file.

Reply:
Done!